root/programs/C#/Frameworks/Infrastructure/Public/Util/Latebind.cs

```diff
@@ -84,8 +85,7 @@ public static object InvokeMethod(string assemblyName, string className, string
             {
                 // アセンブリ ロード
                 string ext = ".dll";
-                int extLen = ext.Length;
-                if (assemblyName.Substring(assemblyName.Length - extLen, extLen) == ext)
+                if (assemblyName.EndsWith(ext))
```

Comment:
@yusukemaegawa 

I see.

@kishoreandukuri 

Please modify the nishino branch directly.
You do not need to use pull request for fix in this time.